docs/_source/community/developer_docs.md

```diff
@@ -98,67 +105,86 @@ The HTML pages are in docs\_build\html.
 [I 231024 11:00:53 server:358] Shutting down...
 ```
 
->**Troubleshooting**
->If you get warnings while building documentation then you can handle them this way:
+> **Troubleshooting**
+> If you get warnings while building documentation then you can handle them this way:
 >
->- If they are toctree or title underline warnings then they can be ignored.
->- If they are import errors then they can be resolved by reinstalling autodoc and argilla from docs/_source/requirements.txt
+> - If they are toctree or title underline warnings then they can be ignored.
+> - If they are import errors then they can be resolved by reinstalling autodoc and argilla from docs/\_source/requirements.txt
 
+## Set up the Development Environment
 
-## Set up the Python Environment
+To work and develop for the core product of Argilla, you need to have all of Argilla's subsystem correctly running. In this section, we'll show how to install the Argilla package, the databases, the frontend and the server.
```

Comment:
Maybe mention the frontend is optional if you want to use the UI but otherwise you can skip the step?